### DISCUSSION TITLE
Carpetright Administration

### DISCUSSION THREAD

**Discussion by TheUKHighStreet** [alltheplaces/alltheplaces#10108] [General]
Carpetright went into administration on 12 July. It has been bought by Tapi carpets. All but 54 stores have closed - the spider is still showing 304 stores open.

**Comment by CloCkWeRX** [comment]
https://en.m.wikipedia.org/wiki/Carpetright

In July 2024, Carpetright announced it had put itself up for sale, appointing PwC to launch a formal sale process.[17] The firm was purchased in the same month by Tapi Carpets, in a pre-packaged insolvency deal, a transaction that is often associated with the new company shedding its commitments to staff and creditors.[18][19][20]

As part of the acquisition, Tapi Carpets would acquire 54 stores, along with 300 jobs and 2 warehouses. The business outside the UK (which would be kept by Nestware Holdings), 218 stores and the company's headquarters in Purfleet would not be part of the deal, affecting more than 1,000 jobs.[21][22]

**Comment by CloCkWeRX** [comment]
From reading the above, let's revisit in a few months and see if the 54 stores rebrand or something else

**Reply by CloCkWeRX** [reply]
Oops: https://github.com/alltheplaces/alltheplaces/pull/10111 exists

**Reply by TheUKHighStreet** [reply]
Thanks (and to @Cj-Malone for fixing it so quickly!). I'm sure you're right that it will probably rebrand these stores eventually.
